diff --git a/lib/spack/spack/util/gpg.py b/lib/spack/spack/util/gpg.py
index 9496e6b063..3481e49163 100644
--- a/lib/spack/spack/util/gpg.py
+++ b/lib/spack/spack/util/gpg.py
@@ -375,7 +375,7 @@ def _socket_dir(gpgconf):
 
     # If there is no suitable gpgconf, don't even bother trying to
     # pre-create a user run dir.
-    if not gpgconf:
-        return None
+    #if not gpgconf:
+    return None
 
     result = None
