diff --git a/lib/spack/spack/store.py b/lib/spack/spack/store.py
index b0a418c..79178fb 100644
--- a/lib/spack/spack/store.py
+++ b/lib/spack/spack/store.py
@@ -119,6 +119,9 @@ def parse_install_tree(config_dict):
             msg = "Cannot pad %s to %s characters." % (root, padded_length)
             msg += " It is already %s characters long" % len(root)
             tty.warn(msg)
+        
+        #BJV - this fix resolves issues with relocation RPATHs
+        root = root.rstrip(os.path.sep)
     else:
         root = unpadded_root
 
